chromadb/proto/utils.py

```diff
@@ -0,0 +1,26 @@
+import json
+from typing import Any, List, Tuple
+
+
+def get_default_grpc_options() -> List[Tuple[str, Any]]:
+    service_config_str = json.dumps(
+        {
+            "methodConfig": [
```

Comment:
should we make this config? also ok with leaving that out for now tbh

Reply:
by config do you mean allowing the retry policy to be configurable? I think it's probably ok to stick with the defaults across all services for now

Reply:
yep just mean putting it in settings, i think its fine for now, just flagging as future work we will likely end up doing